sdk/storage/azure-storage-common/CMakeLists.txt

```diff
@@ -58,7 +58,8 @@ else()
 endif()
 
 # coverage. Has no effect if BUILD_CODE_COVERAGE is OFF
-create_code_coverage(storage azure-storage-common azure-storage-test)
+# excluding json from coverage report
```

Comment:
```suggestion
# Excluding 3rd party JSON library from coverage report.
```

Reply:
If it ships in our source we need to test it.